src/cript/nodes/util/__init__.py

```diff
@@ -286,7 +286,7 @@ def _is_node_field_valid(node_type_list: list) -> bool:
     """
 
     # TODO consider having exception handling for the dict
-    if isinstance(node_type_list, list) and len(node_type_list) == 1 and isinstance(node_type_list[0], str) and node_type_list[0]:
+    if isinstance(node_type_list, list) and len(node_type_list) == 1 and isinstance(node_type_list[0], str) and len(node_type_list[0]) > 0:
```

Comment:
I would write this instead as `node_type_list[0]` because if it is full then it would be truthy and if empty it is false

Reply:
It’s testing if the string isn’t enpty